content/06.model-complexity.md

```diff
@@ -1,34 +1,34 @@
-Combining these three methods resulted in better performance than other methods when implemented on the silver standard dataset in isolation.
-Other approaches like inverse class weighting, oversampling (with methods like Synthetic Minority Oversampling Technique, abbreviated SMOTE), or cascade learning approaches that adequately represent the rarer classes are also beneficial in increasing stability of predictions while accommodating for class imbalance.
+Combining these three methods resulted in better performance than other methods when implemented on a silver standard dataset in isolation.
+Other approaches like inverse class weighting, oversampling (with methods like Synthetic Minority Oversampling Technique, abbreviated SMOTE), or methods that adequately represent the rarer classes are also beneficial in increasing stability of predictions while accommodating for class imbalance.
 
-Presence of label-noise and sparsity in the data can lead to overfitting of models to the training data, such that the models show high prediction accuracy on training data but low prediction accuracy on encountering new data points. 
-Features identified as predictive by overfit models are often misrepresented in the training dataset making the model not generalizable to new data. [@doi:10.1073/pnas.1900654116]
-Regularization can not only protect ML models from poor generalizability caused by overfitting, but also reduce model complexity by reducing the feature space available for training. (Figure[@fig:2]C)
-Regularization is often used in rare variant discovery and immune cell signature discovery studies which are useful examples that also need to accommodate sparsity of data like rare diseases.
+Presence of label-noise and sparsity in the data can also lead to overfitting of models to the training data, such that the models show high prediction accuracy on training data but low prediction accuracy on encountering new data points. 
+Features identified as predictive by overfit models are often not representative of the data making the model not generalizable to new data. [@doi:10.1073/pnas.1900654116]
+In such cases, regularization can not only protect ML models from poor generalizability caused by overfitting, but also reduce model complexity by reducing the feature space available for training. (Figure[@fig:2]C)
+Regularization is often used in rare variant discovery and immune cell signature discovery studies which are useful examples that also need to accomodate sparsity of data like rare diseases.
 For example, among various regularization methods (ridge regression, LASSO regression, and elastic-net)[@doi:10.1111/j.1467-9868.2005.00503.x], hybrid applications of LASSO for capturing combinations of rare variants have proven beneficial [@doi:10.1016/j.ajhg.2008.06.024; @doi:10.1186/1753-6561-5-s9-s113] .
```

Comment:
This section gets jargon-y real fast but is also fairly repetitive across these four sentences. I think this could use an intro sentence describing what regularization is, listing these examples. Then a sentence describing selected applications of these methods in rare disease. Then a sentence saying - you should consider these methods too (basically the last sentence, but it could be shorter).